lib/node_modules/@stdlib/math/base/special/log2/README.md

```diff
@@ -104,6 +104,95 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/log2.h"
+```
+
+#### stdlib_base_log2( x )
+
+Evaluates the binary logarithm.
+
+```c
+double out = stdlib_base_log2( 4.0 );
+// returns 2.0
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+double stdlib_base_log2( const double x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/log2.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+int main( void ) {
+    double x;
+    double v;
+    int i;
+    
```

Comment:
```suggestion

```

I suggest configuring your editor to automatically remove trailing spaces. You should have already installed EditorConfig (https://github.com/stdlib-js/stdlib/blob/58c9a5053430b55c974e9ecccfc45c51c3fefc9e/.editorconfig#L28).